graph/client.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function, unicode_literals
+from PyInquirer import style_from_dict, Token, prompt, Separator
```

Comment:
Let's aim to make this optional, so even people without `PyInquirer` who are offline, unable to install it, can use the tool.